frontend/awx/views/schedules/hooks/scheduleHelpers.tsx

```diff
@@ -40,6 +41,84 @@ export const scheduleResourceTypeOptions: string[] = [
   'projects',
 ];
 
+export function useGetScheduleUrl() {
+  const { t } = useTranslation();
+  return useCallback(
+    (route: string, schedule: Schedule) => {
+      const unified_job_type = schedule.summary_fields.unified_job_template.unified_job_type;
+      const params = {
+        id: schedule.summary_fields.unified_job_template?.id.toString(),
+        schedule_id: schedule.id.toString(),
+      };
+
+      if (unified_job_type === 'inventory_update') {
+        const inventory_update_params = {
```

Comment:
generally in JS/TS land we use camel case for variable names
```suggestion
        const inventoryUpdateParams = {
```